analyzer/src/funTest/assets/projects/synthetic/dotnet-direct-dependencies-only-expected-output.yml

```diff
@@ -68,25 +129,20 @@ packages:
 - id: "NuGet::System.Threading:4.0.11"
   purl: "pkg:nuget/System.Threading@4.0.11"
   authors:
-  - "Microsoft"
+  - "Microsoft <>"
   declared_licenses:
-  - "http://go.microsoft.com/fwlink/?LinkId=329770"
+  - "LicenseUrl: http://go.microsoft.com/fwlink/?LinkId=329770"
   declared_licenses_processed:
-    spdx_expression: "LicenseRef-scancode-ms-net-library-2019-06"
-    mapped:
-      http://go.microsoft.com/fwlink/?LinkId=329770: "LicenseRef-scancode-ms-net-library-2019-06"
+    unmapped:
+    - "LicenseUrl: http://go.microsoft.com/fwlink/?LinkId=329770"
   description: "Provides the fundamental synchronization primitives, including System.Threading.Monitor\
-    \ and System.Threading.Mutex, that are required when writing asynchronous code.\n\
```

Comment:
There are two fields upstream: summary and description. We use only the summary.